tide_landing_page.install

```diff
@@ -2669,3 +2669,26 @@ function tide_landing_page_update_8051() {
     ->load('paragraph.statistics_grid.default');
   $entity_form_display->setComponent('field_statistic_block', $config_read['content']['field_statistic_block'])->save();
 }
+
+/**
+ * Removes Card event and Card event automated paragraph types.
+ */
+function tide_landing_page_update_8052() {
+  $paragraph_types = [
+    'card_event',
+    'card_event_auto',
```

Comment:
What about these card types, news_listing, card_promotion, card_promotion_auto. These are deprecated as well and if there should be removed.

Reply:
hey @MdNadimHossain 
In your branch, have you removed all the relevant fields? We can use your branch instead. I'll close off this branch.

Reply:
your branch is ready to merge? @MdNadimHossain

Reply:
I think I did. But it needs to be updated with the latest develop and also testing required

Reply:
> your branch is ready to merge? @MdNadimHossain

nope, it is not ready to merge, it was on hold for a long time because someone from the content team suppose to confirm things. So it is long outdated now

Reply:
right, let's merge this one specifically for the  https://digital-vic.atlassian.net/browse/SDPAP-7408 ticket.

Reply:
Cool